ajax/libs/vis-timeline/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vis-timeline",
+  "npmName": "vis-timeline",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
```
node_modules/vis-timeline/dist
├── types
│   └── index.d.ts
├── vis-timeline-graph2d.css
├── vis-timeline-graph2d.esm.js
├── vis-timeline-graph2d.esm.js.map
├── vis-timeline-graph2d.min.css
├── vis-timeline-graph2d.min.js
└── vis-timeline-graph2d.min.js.map
```

`types` directory is missing from the files you've included. If it isn't needed, this glob should just be `*` instead of `**/*`.